src/Tizen.NUI/src/public/WindowSystem/TzShell.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
```

Comment:
Please add Apache 2.0 license boilerplate to top of all files.

Reply:
I added it